### ISSUE TITLE
1.3.2. Advisory Technique recommends using justified text. Shouldn't it be left/right-aligned?

### ISSUE THREAD

**Issue by jfhector** [w3c/wcag#697]
One of the advisory techniques for 1.3.2 is "Using left-justified text for languages that are written left to right and right-justified text for languages that are written right-to-left (future link)".

I'm surprised that it says "left-justified" and "right-justified". I believe that, for people with dyslexia, justified text is harder to read than left-aligned or right-aligned text (depending on the language's reading direction).

Is this a mistake, or am I missing something?

Note: This advisory technique is listed in the "How to Meet WCAG 2 (Quick Reference)" document under SC 1.3.2. But it is not listed in the "Understanding document" for SC 1.3.2.

**Comment by mraccess77** [comment]
My personal read is that left justified and left aligned are the same.  This is opposed to "full justification" which is harder for some to read.

**Comment by Myndex** [comment]
Yes, "left justified" means all lines start aligned on the left, the terms left aligned and left justified are synonymous.

"Full" justified means both the left and right of each sentence in a text column are aligned. In order to do this, the text of each line has adjustments made to kerning/tracking, hyphenation, and glyph (letter) width among other things. The result ends up making the text harder to read, so common practice is to avoid full justification.

For text that reads left to right, the best practice is left justified/ragged right. 

I'm assuming the confusion comes from CSS using the term "justify" for full justification, and left, right, and center for the other alignments:

  text-align: center
  text-align: left
  text-align: right
  text-align: justify

### On two related notes:
**Flowing:** as full justified may make a page "look nicer" at the expense of readability, so does flowing text. While it might look nice from a design standpoint to flow text around a curved object it makes the text much harder to read. Best practice (my opinion) is to flow only on the right of a column, leaving the left justified, and never ever allow a flow to break a sentence in the middle and continue on the right side of an object.

**Hyphenation:** As a personal preference, when I layout a document I set hyphenation rules very strict to avoid hyphenation whenever possible. As a reader I find hyphens distracting, so I avoid them as a typographer.

**Comment by alastc** [comment]
> I believe that, for people with dyslexia, justified text is harder to read than left-aligned or right-aligned text (depending on the language's reading direction).

Indeed, but as the others have said the meaning of 'left-justified' is that it is left aligned.

> This advisory technique is listed in the "How to Meet WCAG 2 (Quick Reference)" document under SC 1.3.2. But it is not listed in the "Understanding document" for SC 1.3.2.

Thanks for spotting that, I'll check with @michael-n-cooper as I think the page-build process might be skipping things.

**Comment by michael-n-cooper** [comment]
I can't tell from context what you think might be getting skipped by the build. But one thing to check is, text not inside a <p> element is likely to get skipped. I frequently see <section><h2>heading</h2>text</section> and the "text" in that scenario isn't processed, the generator looks only at element children of <section> for various specific reasons, it ignores text node children of <section>.

**Comment by patrickhlauke** [comment]
>This advisory technique is listed in the "How to Meet WCAG 2 (Quick Reference)" document under SC 1.3.2. But it is not listed in the "Understanding document" for SC 1.3.2.

should check if this is still the case, and fix this aspect if so

**Comment by giacomo-petri** [comment]
Obsolete. It no longer appears in either Understanding or Quick Ref:
- Understanding: https://www.w3.org/WAI/WCAG22/Understanding/meaningful-sequence.html
- Quick Ref: https://www.w3.org/WAI/WCAG22/quickref/?versions=2.2&showtechniques=132#qr-content-structure-separation-sequence

I propose we close this.

Note: It was listed in WCAG 2.0 as an Advisory Technique (without a reference link): https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-sequence.html
However, it is also absent from Quick Ref for WCAG 2.0: https://www.w3.org/WAI/WCAG22/quickref/?versions=2.0&showtechniques=132#qr-content-structure-separation-sequence

**Comment by mbgower** [comment]
If you have concerns, please reopen.
